test cases/objc/2 nsstring/meson.build

```diff
@@ -2,6 +2,8 @@ project('nsstring', 'objc')
 
 if host_machine.system() == 'darwin'
   dep = dependency('appleframeworks', modules : 'foundation')
+elif host_machine.system() == 'cygwin'
+  error('MESON_SKIP_TEST GNUstep isn't packaged for Cygwin.')
```

Comment:
So it does raise an exception, but because the output contains `MESON_SKIP_TEST`, meson just marks it as skipped. Please fix the typo. ;)